tests/extrema_test.py

```diff
@@ -86,9 +86,12 @@ def test_error_handling(self):
         with self.assertRaises(TypeError):
             ak.mink(list(range(0, 10)), 1)
 
-        with self.assertRaises(TypeError):
-            ak.mink(testArray, "1")
-
+        with self.assertRaises(TypeError) as cm:
+            ak.mink(testArray, '1')
+        self.assertEqual(('type of argument "k" must be one of (int, int8, int16, int32, ' +
```

Comment:
Just marking this in one spot. I know @pierce314159 and I have had issues in the past when we check the specific error message. I think in general we have been assuming the correct error being raised is enough.

Reply:
Yeah I'm of the opinion that it's better not the check the exact error message. We've seen strangeness before where users have seen test failures when there's nothing wrong

There was a case where where we were checking for:
`'type of argument "bins" must be one of (int, int64, uint64); got str instead'`
and users were getting errors cause there's said:
`'type of argument "bins" must be one of (int, np.int64, uint64); got str instead'`
https://github.com/Bears-R-Us/arkouda/issues/1105

So I think just verifying it's a `TypeError` should be enough

Reply:
@Ethan-DeBandi99 @hokiegeek2 removed error message checking